### ISSUE TITLE
Text "decorations" (`\cancel`, `\underbrace`, etc.) are sized incorrectly for Unicode text

### ISSUE THREAD

**Issue by benchristel** [mathjax/MathJax#3056]
### Issue Summary

The "decorations" created by e.g. `\cancel`, `\underbrace`, and `\overline` are sized incorrectly when the command's argument contains unicode text.

### Steps to Reproduce:

1. Go to https://www.mathjax.org/#demo
2. Enter `$\underbrace{\text{你好}}$` in the box

### Technical details:

* MathJax Version: 3.2.2
* Client OS: macOS 12.6
* Browser: Microsoft Edge 113

### Supporting information:

<img width="1034" alt="screenshot showing the problem" src="https://github.com/mathjax/MathJax/assets/693920/137997ce-4ad9-463b-8ca9-50f8e7d79b5f">

**Comment by dpvc** [comment]
I am not able to reproduce your results using Edge 113.0.1774.57 on Mac OS 12.6.5.  Here is what I see:

<img width="450" alt="Edge" src="https://github.com/mathjax/MathJax/assets/432782/01f658a2-daff-4834-bdb4-a93d47811f79">

Have you made any changes using the MathJax contextual menu that might be affecting the results?

**Comment by benchristel** [comment]
Yes, I had `{"scale":"1.8"}` in `MathJax-Menu-Settings`. Sorry for the confusion.

**Comment by dpvc** [comment]
Well, it should still work with that setting, so there may still be a problem with Edge, though I'm still not able to reproduce it.  I will leave this open to check into in the future.
